tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -26,12 +26,10 @@ data. We will also learn how to perform some basic raster calculations
 
 After completing this activity, you will be able to:
 
-* Extract a "slice" of data from a hyperspectral data cube.
-* Create a rasterstack in R which can then be used to create RGB images from 
-bands in a hyperspectral data cube.</li>
+* Extract a "slice" of data from a hyperspectral data cube. </li>
```

Comment:
Do not need the html code for line endings, remove.

Reply:
Done.